Tests/BootCamp.Chapter.Tests/ConsoleStub.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter.Tests
+{
+    public static class ConsoleStub
+    {
+        public static StringWriter StubConsole(string readLineReturn)
+        {
+            var output = new StringWriter();
+            Console.SetOut(output);
+
+            var input = new StringReader(readLineReturn);
+            Console.SetIn(input);
+
+            return output;
+        }
+    }
+}
```

Comment:
This file has not the same folder structure as in the original repository. This shouldn't be a problem as long as we don't merge it in but it would defenitly cause problems in form of code duplication or more.

Reply:
Yes, this was supposed to be in utilities. Why did you move the file, I wonder :) ?

Reply:
I hadn't cloned the Lesson3 branch so I didn't have the original files to work with.  I hadn't paid attention to the fact that I didn't have a "Utils" folder as well.